docs/glossary.md

```diff
@@ -24,3 +24,27 @@ manageability created need for BMC (Board Management
 Controllers), EC (Environmental Controllers) and even more
 specialized one like USB Power Delivery firmware. What may
 make that firmware also covered by above definition.
+
+## Dasharo Hardware Compatibility List Report
```

Comment:
@pietrushnic should we point to this section of glossary while printing disclaimer in dts? https://github.com/Dasharo/meta-dts-ce/blob/main/meta-dts-distro/recipes-dts/dts/files/dts#L22

Reply:
We can, but link would be relatively long. Maybe just refer to Glossary at docs.dasharo.com - TBH we should have way more extensive section related to HCL to prove value we get from logs we received, but for now just reference to website is enough.

Reply:
Ok, the reference to the Glossary will be added while working on issue https://github.com/Dasharo/dasharo-issues/issues/92#issuecomment-1288804122